ctapipe/irf/irf_classes.py

```diff
@@ -0,0 +1,74 @@
+"""
+Define a parent IrfTool class to hold all the options
+"""
+from enum import Enum
+
+import astropy.units as u
+from pyirf.cuts import calculate_percentile_cut
+from pyirf.spectral import CRAB_HEGRA, IRFDOC_ELECTRON_SPECTRUM, IRFDOC_PROTON_SPECTRUM
+
+from ..core import Component
+from ..core.traits import Float, Integer
+
+
+class Spectra(Enum):
+    CRAB_HEGRA = 1
+    IRFDOC_ELECTRON_SPECTRUM = 2
+    IRFDOC_PROTON_SPECTRUM = 3
+
+
+PYIRF_SPECTRA = {
```

Comment:
I'd just call this "spectra" because we might define our own using the base classes from pyirf